store/src/java/com/zimbra/cs/service/util/JWTUtil.java

```diff
@@ -126,7 +129,7 @@ public static final String getSalt(Element soapCtxt, Map engineCtxt) {
                     javax.servlet.http.Cookie cookies[] =  req.getCookies();
                     if (cookies != null) {
                         for (int i = 0; i < cookies.length; i++) {
-                            if (cookies[i].getName().equals(Constants.ZM_JWT_COOKIE)) {
+                            if (ZimbraCookie.COOKIE_ZM_JWT.equals(cookies[i].getName())) {
```

Comment:
Don't we need to check the the value of cookies[x].getName() in case-insensitively?